File: fast-react/src/common/enmus/demand-trace-dealstate-enum.js
export const demandTraceDealstateEnum = {
    NEW_ADD: 1,
    WORKING: 2,
    HUNG_UP: 3,
    FINISHED: 4,
    REPEAL: 5,
    POSTPONE: 6,
    getName: (code) => {
        if (code ===1) return '新增'
        if (code === 2) return '进行中'
        if (code === 3) return '挂起'
        if (code ===4) return '已完结'
        if (code === 5) return '作废'
        if (code === 6) return '暂缓'
    }
}

File: fast-boot/src/main/java/com/example/fastboot/common/enums/DemandTraceDealStateEnum.java
package com.example.fastboot.common.enums;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.Getter;

/**
 * @Author: LiuZhaobo
 * @Date: 2021/10/17/9:52
 */
@Getter
@AllArgsConstructor
public enum DemandTraceDealStateEnum {
    NEW_ADD(1, "新增"),//缺省是0
    WORKING(2, "进行中"),
    HUNG_UP(3, "挂起"),
    FINISHED(4, "已完结"),
    REPEAL(5, "作废"),
    POSTPONE(6, "暂缓");
    private int code;
    private String name;
}
